CONTRIBUTING.md

```diff
@@ -1,81 +1,104 @@
-	   ```
-	   $ pack_model.sh --lm <language model> <tr_conf> <dec_conf> <cmvn> <e2e>
-	   ```
-	 - `pack_model.sh` also produces a packed espnet model (`model.tar.gz`). If you upload this model to somewhere with a download link,
-	   please put the link information to `RESULTS.md`.
-	 - please contact Shinji Watanabe <shinjiw@ieee.org> if you want a web storage to put your model files.
-
-Once you finish implementing a feature or bugfix, please send a Pull Request to https://github.com/espnet/espnet
+To make a report for `RESULTS.md`
+ - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
+   You'll get environmental information and the evaluation result of each experiment in a markdown format.
+   ```
+   $ show_result.sh
+   ```
+ - execute `pack_model.sh` at the main directory of a recipe as follows. You'll get model information in a markdown format
+   ```
+   $ pack_model.sh --lm <language model> <tr_conf> <dec_conf> <cmvn> <e2e>
+   ```
+ - `pack_model.sh` also produces a packed espnet model (`model.tar.gz`). If you upload this model to somewhere with a download link,
+   please put the link information to `RESULTS.md`.
+ - please contact Shinji Watanabe <shinjiw@ieee.org> if you want web storage to put your model files.
+
+#### 1.3.2 ESPnet2 recipes
+
+#### 1.3.3 Typical issues in recipes
+
+## 2 Pull Request
+Once you finish implementing a feature or bug-fix, please send a Pull Request to https://github.com/espnet/espnet
 
 If you are not familiar with creating a Pull Request, here are some guides:
 
 - http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
 - https://help.github.com/articles/creating-a-pull-request/
 
-## Version policy and development branches
+## 3 Version policy and development branches
 
 We basically develop in the `master` branch.
 
 1. We will keep the first version digit `0` until we have some super major changes in the project organization level.
 
-2. The second version digit will be updated when we have major updates including new functions and refactoring, and 
+2. The second version digit will be updated when we have major updates, including new functions and refactoring, and 
    their related bug fix and recipe changes.
-   This version update will be done roughly at every half year so far (but it depends on the development plan).
+   This version update will be done roughly every half year so far (but it depends on the development plan).
 
-3. The third version digit will be updated when we fix serious bugs or accumulate some minor changes including
+3. The third version digit will be updated when we fix serious bugs or accumulate some minor changes, including
    recipe related changes periodically (every two months or so).
 
-## Unit testing
+## 4 Unit testing
 
 ESPnet's testing is located under `test/`.  You can install additional packages for testing as follows:
 ``` console
 $ cd <espnet_root>
 $ pip install -e ".[test]"
```

Comment:
```suggestion
$ cd <espnet_root>
$ source tools/venv/bin/activate
$ pip install -e ".[test]"
```

May be better to install in virtual env. I dunno for others but in my case I can't install locally test packages because of the python2/3 versions.

Reply:
very helpful suggestion! I think we might follow `activate_python.sh` instead of venv to keep the test in update-to-data versions.